apps/remix-ide/src/blockchain/blockchain.js

```diff
@@ -501,7 +501,7 @@ class Blockchain extends Plugin {
       if (isVM) {
         const hhlogs = await this.web3().eth.getHHLogsForTx(txResult.transactionHash)
         if (hhlogs && hhlogs.length) {
-          let finalLogs = 'console.log:\n'
+          let finalLogs = '<b>console.log:</b>\n'
```

Comment:
would you do the same for the unit tests log?

Reply:
It is done there before here: https://github.com/ethereum/remix-project/blob/master/apps/remix-ide/src/app/tabs/test-tab.js#L184